lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -639,9 +639,26 @@ function _dosomething_user_send_to_message_broker() {
     'user_country'      => isset($user_country_code) ? $user_country_code : 'US',
   );
 
-  // 26+ Club: Override Mobile Commons.
+  // Subscribe new members to Do Something Master Campaign.
   $mobile = dosomething_user_get_field('field_mobile', $account);
-  $opt_in = variable_get('dosomething_signup_mobilecommons_opt_in_path_user_register');
+
+  // Load opt-in path for user country.
+  // Note: only US messaging is supported now.
+  // Uncomment this code when more contries are supported.
+  // See https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_signup/dosomething_signup.admin.inc#L43
+
+  /*
+  $user_language = $user->language;
+  $country_code = dosomething_global_convert_language_to_country($user_language);
+  if (empty($country_code)) {
+    $country_code = 'global';
+  }
+  */
+
+  // Only US is supported.
+  $country_code = 'US';
```

Comment:
This variable is exist only for us users now. The code above is commended because it's a reminder how to make this when this is available not for `US` user only.